test/test_valgrind.sh

```diff
@@ -188,7 +188,21 @@ echo
 echo "======================================================================"
 echo
 
-for log in $(ls -1 ${PATH_TESTS}.log ${PATH_EXAMPLES}.log); do
+LOG_FILES=""
+NT=$(ls -1 ${PATH_TESTS}.log 2>/dev/null | wc -l)
+if [ $NT -gt 0 ]; then
+	LOG_FILES="$LOG_FILES $(ls -1 ${PATH_TESTS}.log | xargs)"
+fi
+NE=$(ls -1 ${PATH_EXAMPLES}.log 2>/dev/null | wc -l)
+if [ $NE -gt 0 ]; then
+	LOG_FILES="$LOG_FILES $(ls -1 ${PATH_EXAMPLES}.log | xargs)"
+fi
+if [ $(($NT + $NE)) -eq 0 ]; then
```

Comment:
having these numbers you could already check if `$NT + $NE == $ANY_TEST_FAILED`, I guess

Reply:
Added below.
Done.